tooling/compose-intellij-platform/sample/plugin-2/build.gradle.kts

```diff
@@ -0,0 +1,22 @@
+import org.jetbrains.compose.compose
+
+plugins {
+    id("org.jetbrains.kotlin.jvm")
+    id("org.jetbrains.intellij")
+    id("org.jetbrains.compose")
+}
+
+jvmTarget("11")
+
+dependencies {
+    compileOnly(compose.desktop.currentOs)
+}
+
+intellijPlugin(group = "org.jetbrains.compose.intellij.platform.sample.plugin2") {
+    plugins.set(
```

Comment:
Would be great to see how out of project configuration would look here? Do we need to publish an additional plugin, right?

Reply:
I don't understand the question